### DISCUSSION TITLE
fraction is too small in inline math mode

### DISCUSSION THREAD

**Discussion by mojtaba82** [typst/typst#3801] [General]
### Description

Fraction is too small in inline math mode.  In block mode  is great. Late has \dfrac macro that type fraction in inline mode as large as block mode.
This image is rendered by Typst  0.11 by bot of discord.
![image](https://github.com/typst/typst/assets/4151722/aa7f8d4e-f56c-4322-8837-57737989a963)

### Operating system

Linux

### Typst version

- [X] I am using the latest version of Typst

**Comment by wrzian** [comment]
You can wrap in `display()` to get back to display mode math:
```
Compute $display((3x^2+9)/(y dot y -5x))$
 for $x=-3$ and $y=4$
$ (3x^2+9)/(y dot y -5x) $
```
![image](https://github.com/typst/typst/assets/133046678/71061c4d-56c5-4885-b851-375e489f7c5d)
